supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -165,17 +143,17 @@ Do not use "Firewire" or "firewire." Although FireWire is a trademark of Apple C
 
 *Use it*: yes
 
-*Incorrect forms*: 
+*Incorrect forms*:
 
 *See also*: xref:futex[futex], xref:mutexes[mutexes]
 
 [discrete]
 [[fuzzy]]
 ==== fuzzy (adjective)
-*Description*: It is only correct to use "fuzzy" as adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and examples. 
+*Description*: It is only correct to use "fuzzy" as adjective when referring to "fuzzy searches" (the technique of finding strings that match a pattern approximately, rather than exactly). See http://www.stylepedia.net/#chap-Red_Hat_Technical_Publications-Writing_Style_Guide-Avoiding_Slang_Metaphors_and_Misleading_Language[Avoiding Slang, Metaphors, and Misleading Language] for details and examples.
```

Comment:
Maybe add "an" before adjective.

Reply:
@leswilliams44 This PR is just removing/properly alphabetizing entries. I didn't modify the content at all, so I'd probably prefer to keep it that way here. Would you mind following up with a separate PR to fix these minor updates?

Reply:
sure thing

Reply:
Thanks!